recipes/ptxcompiler/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "ptxcompiler" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rapidsai/ptxcompiler/archive/refs/tags/{{ version }}.tar.gz
+  sha256: c7021a0946478404c4055a612fd725af95d53ec23eb3dd8d8bc0b5ac55eb44d0
+  fn: ptxcompiler.tar.gz
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  skip: true  # [cuda_compiler_version == "None"]
+  skip: true  # [cuda_compiler_version == "10.2"]
+  skip: true  # [cuda_compiler_version == "11.0"]
+  script_env: 
+    - CUDA_HOME  # [ppc64le or aarch64]
+    - LIBRARY_PATH=/usr/local/cuda/lib64    
```

Comment:
It seems that this patch:
https://github.com/rapidsai/ptxcompiler/commit/5766ac29aab9ddba3a160d45d5d86cfbd3c9a647
was recently submitted, to help you avoid this export.

Can you cut a new release and try building without this customization.